### ISSUE TITLE
Is there a way to emulate a dashboard like view with a navigator?

### ISSUE THREAD

**Issue by desarrollo03TR** [ComponentFactory/Krypton#73]
I need a way to show several non resizable small windows just like a dashboard. I think that a navigator is the right component.

**Comment by Smurf-IV** [comment]
Non-Resizeable, just means that the borders of the controls are not "Related" (See properties of each child control) to more than 2 sides of the parent control.. Not really a Krypton specific thing, unless I'm missing something !

**Comment by lrhage** [comment]
Not strictly a krypton specific thing, but I think it is easier to create a dashboard using a krypton workspace/navigator.

What I need is something like this:


![](https://dpspowerbi.blob.core.windows.net/powerbi-prod-media/powerbi.microsoft.com/en-us/documentation/articles/powerbi-content-pack-microsoft-dynamics-nav/20170906112627/dashboard.png)

**Comment by FreeFlyingEnchilada** [comment]
Is there any specific behavior you're after? If not, it seems like just a regular old TableLayoutPanel with native drag and drop would more than suffice to create this.

**Comment by lrhage** [comment]
Specific behavior? not really, just adding small windows to a big control, with possibility to mazimize/rearrange/close, and with an specific size.
